test/e2e/framework/util.go

```diff
@@ -4396,7 +4396,7 @@ func OpenWebSocketForURL(url *url.URL, config *restclient.Config, protocols []st
 	if err != nil {
 		return nil, fmt.Errorf("failed to create tls config: %v", err)
 	}
-	if tlsConfig != nil {
+	if tlsConfig != nil || url.Scheme == "https" {
 		url.Scheme = "wss"
 		if !strings.Contains(url.Host, ":") {
```

Comment:
this should just be `if url.Port() == ""`

Reply:
Actually it's entirely unnecessary. https://github.com/kubernetes/kubernetes/blob/91d2bd3951f3a2e7290ef0cf5ec546ba228944d7/vendor/golang.org/x/net/websocket/client.go#L66-L78

Reply:
Do you want me to change it in this PR?

Reply:
Sure, since you are already around might as well clean things up. Thanks!

Reply:
I removed it!